### PR TITLE
`relational.write`: mysql support

### PR DESCRIPTION
**Pull request by spicy-sauce**

### CODE REVIEW

File: core/src/datayoga_core/blocks/relational/write/block.py [hunk 1]
```diff
@@ -7,17 +8,38 @@
...
 
 
+@unique
+class DbType(Enum):
+    MYSQL = "mysql"
+    PSQL = "postgresql"
+
+
+def get_driver_name(db_type: str) -> str:
+    if db_type == DbType.MYSQL.value:
```
Comment (zalmane) [inline core/src/datayoga_core/blocks/relational/write/block.py:23]: can we turn this into a dict and just pull from there? we will have tens of them

File: core/src/datayoga_core/blocks/relational/write/block.py [hunk 2]
```diff
@@ -28,9 +50,9 @@ def init(self, context: Optional[Context] = None):
         logger.debug(f"Initializing {self.get_block_name()}")
 
         connection = utils.get_connection_details(self.properties.get("connection"), context)
-        db_type = connection.get("type")
+        db_type = connection.get("type").lower()
         engine_url = sa.engine.URL.create(
-            drivername=db_type,
+            drivername=get_driver_name(db_type),
```
Comment (zalmane) [inline core/src/datayoga_core/blocks/relational/write/block.py:55]: this should just be a default. we should support passing the driver name explicitly as well

File: core/src/datayoga_core/resources/schemas/connections.schema.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
             "type": {
               "description": "DB type",
               "type": "string",
-              "pattern": "^(?!redis|cassandra$)"
+              "enum": ["mysql", "postgresql"]
```
Comment (zalmane) [inline core/src/datayoga_core/resources/schemas/connections.schema.json:15]: what about cassandra and redis?

**Review by zalmane — CHANGES_REQUESTED** [review]

File: core/src/datayoga_core/resources/schemas/connections.schema.json (hunk 3 above)
Comment (spicy-sauce) [reply core/src/datayoga_core/resources/schemas/connections.schema.json:15]: they have their own section in the connections, here we used to consider everything that it's not redis and cassandra as `relational.write`, I decided to list the databases we actually support.

**Review by zalmane — APPROVED** [review]

**Pull request merged**